galasa-parent/galasa-boot/src/main/java/dev/galasa/boot/Launcher.java

```diff
@@ -561,4 +565,21 @@ private void commandLineError(String message) {
                         + "Example Resource Management arguments: --obr infra.obr --obr test.obr --resourcemanagement");
         System.exit(-1);
     }
+
+    public boolean validateJavaLevel(){
+        String version = System.getProperty("java.version");
+        logger.trace("Checking version of Java, found: " + version);
+        if(version == null || version.isEmpty()){
+            logger.error("Unable to determine Java version - will exit");
```

Comment:
If log statements know we'll exit when we return false, seems weird that the System.exit() is done outside the function ?

Reply:
Also, didn't we have a SystemProperty class with an interface we could use for this too ?

Reply:
allowed checkJavaVersion to throw the Exception directly so it's a lot cleaner.  SystemEnvironment is in the prior PR